### ISSUE TITLE
Reference to 'Custom Hero Element Selectors' needs removing from the UI

### ISSUE THREAD

**Issue by Nooshu** [WPO-Foundation/webpagetest#1417]
Functionality has been removed in favor of LCP and element timing. The [documentation related to this setting](https://github.com/WPO-Foundation/webpagetest/blob/master/docs/Metrics/HeroElements.md) can therefor also be removed.

For reference `heroElementTimes` are also [mentioned here](https://github.com/WPO-Foundation/webpagetest/blob/f0577c675149676fdc832976914a0b285db7da83/www/index.php#L171)

**Comment by andydavies** [comment]
@Nooshu When did it get removed? I'm sure I used it in Dec?

**Comment by Nooshu** [comment]
After speaking to @pmeenan last night he mentioned the functionality has now been removed. I conducted a simple test which is what flagged it with for me:

In my simple example I've defined 2 elements on http://www.gov.uk using `{"govukLogo": ".header-logo", "searchButton": ".gem-c-search__submit"}`. But I can't see any reference to them in the results page or in the resulting JSON. [Test seen here](https://www.webpagetest.org/result/201229_Di2N_a34c37e0b957557fc4930cada028c518/)
